lib/util.ex

```diff
@@ -2,11 +2,19 @@ defmodule Util do
   @auth_env_var "SMASH_GG_TOKEN"
   @default_storage_file "tournaments.json"
 
-  def update_tourneys(filename \\ @default_storage_file) do
-    {:ok, tourneys} = get_all_tourneys()
-    tourneys
-    |> JSON.encode!
-    |> then(&File.write!(filename, &1))
+  def update_tourneys do
+    case get_all_tourneys() do
+      {:ok, tourneys} ->
+        try do
+          tourneys
+          |> JSON.encode!
+          |> then(&File.write!(@default_storage_file, &1))
+        rescue
+          e ->  IO.puts("An error occurred while either encoding the tourneys map to JSON or writing to the json file: " <> e.message)
+        end
+      {:error, reason} -> IO.puts(reason)
+    end
+
```

Comment:
making get all tourneys conform to return the :ok or :error case mapping seemed like a good idea here

Reply:
Agreed! Once the pagination mess is eventually taken care of, we might decide we do want to have some top-level prettification of errors or something, who knows.

Reply:
Did we want to add a System.stop here as well?

Reply:
Definitely!